utils/actions/candyRaffle.js

```diff
@@ -0,0 +1,212 @@
+import { ethers } from 'ethers'
+import toast from 'react-hot-toast'
+
+const CONFIRMATIONS = 1
```

Comment:
```suggestion
const CONFIRMATIONS = 2
```
Prevent the caching from the free RPC to screw up things

Reply:
713a24a